### PR TITLE
🆙 Top selling channels

### PR DESCRIPTION
**Pull request by WRadoslaw**
#3779

### CODE REVIEW

File: packages/atlas/src/components/TopSellingChannelsTable/TopSellingChannelsTable.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { TopSellingChannelsTable } from '@/components/TopSellingChannelsTable/TopSellingChannelsTable'
+
+export default {
+  title: 'other/TopSellingChannelsTable',
+  component: TopSellingChannelsTable,
+}
+
+const Template = () => <TopSellingChannelsTable />
```
Comment (attemka) [inline packages/atlas/src/components/TopSellingChannelsTable/TopSellingChannelsTable.stories.tsx:28]: its missing the apollo provider to fetch the sender data

**Review by attemka — CHANGES_REQUESTED** [review]

**Review by attemka — APPROVED** [review]

**Review by drillprop — DISMISSED** [review]
Currently, this table is only available in the storybook, which is not very useful. Can we add this to the marketplace view? I don't think anything blocks us currently from actually using the table.

File: packages/atlas/src/components/TopSellingChannelsTable/TopSellingChannelsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        : data?.topSellingChannels.map((data, index) => ({
+            index: (
+              <Text variant="t100" as="p" color="colorTextMuted">
+                {index + 1}
+              </Text>
+            ),
+            salesVolume: (
+              <JoyAmountWrapper>
+                <JoyTokenIcon variant="gray" />
+                <NumberFormat variant="t200-strong" as="p" value={new BN(data.amount)} margin={{ left: 1 }} />
```
Comment (attemka) [inline packages/atlas/src/components/TopSellingChannelsTable/TopSellingChannelsTable.tsx]: missing `format="short"` here

**Comment by attemka** [comment]
LGTM after one minor fix

**Review by attemka — APPROVED** [review]

**Pull request merged**